src/App.js

```diff
@@ -1,18 +1,22 @@
+import React from "react";
 import TodoList from "./TodoList";
 import AddTodoForm from "./AddTodoForm";
 
 function App() {
-  const NewTodo = (props) => {
-    const [newTodo,setNewTodo] = React.useState('');
-  }
+ 
+
+    const[todoList, setTodoList]= React.useState([]);
+    const addTodo =(newTodo)=>{
+      console.log(newTodo);
+ setTodoList([...todoList,newTodo]);
+      
+    }
```

Comment:
A couple more indentation and spacing errors. Line 11 should be indented, and line 12 shouldn't be blank.